cli/dcos-spark/submit_builder_test.go

```diff
@@ -363,3 +363,33 @@ func (suite *CliTestSuite) TestSaslSecret() {
 
 	suite.checkProps(v, stringProps)
 }
+
+func (suite *CliTestSuite) TestPackagesFlag() {
+	sparkPackages := "group.one.id:artifact-one-id:version.one,group.two.id:artifact-two-id:version.two"
+	inputArgs := fmt.Sprintf(
+		"--packages %s "+
+		"--class %s "+
+		"%s --input1 value1 --input2 value2", sparkPackages, mainClass, appJar)
+
+	cmd := createCommand(inputArgs, image)
+        payload, err := buildSubmitJson(&cmd, marathonConfig)
+
+        jsonMap := make(map[string]interface{})
+
+        json.Unmarshal([]byte(payload), &jsonMap)
+
+        if err != nil {
+                suite.T().Errorf("%s", err.Error())
+        }
+
+        stringProps := map[string]string{
+		"spark.jars.packages": sparkPackages,
```

Comment:
when added, we need to check for `--conf spark.jars.ivy` property presence too

Reply:
Added the required check